src/arrayMethodPop.js

```diff
@@ -5,7 +5,12 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    if (this.length === 0) {
```

Comment:
В булем значении 0 === false, можно просто
```suggestion
    if (!this.length) {
```